tools/mmp/Tuning.cs

```diff
@@ -86,19 +86,34 @@ public static void Process (LinkerOptions options, out MonoMacLinkContext contex
 
 			try {
 				pipeline.Process (context);
-			} catch (AssemblyResolutionException fnfe) {
+			}
+			catch (Exception e) {
```

Comment:
Minor: the closing brace for the `try` block should be on the same line as the `catch`.